src/main/java/cms/gov/madie/measure/services/ElmTranslatorClient.java

```diff
@@ -45,7 +47,21 @@ public boolean hasErrors(ElmJson elmJson) {
     try {
       ObjectMapper mapper = new ObjectMapper();
       JsonNode jsonNode = mapper.readTree(elmJson.getJson());
-      return jsonNode.has("errorExceptions") && jsonNode.get("errorExceptions").size() > 0;
+      boolean parsingErrors = false;
+      if (jsonNode.has("errorExceptions")) {
+        JsonNode errorExceptionsNode = jsonNode.get("errorExceptions");
+        Iterator<JsonNode> entyIter = errorExceptionsNode.iterator();
+        while (entyIter.hasNext()) {
+          JsonNode errorNode = entyIter.next();
+          if (errorNode.get("type") != null
```

Comment:
we are only checking for parsing errors. what are the other types of errors that we want to ignore?

Reply:
@adongare Only parsing errors and externalErrors, to be consistent with frontend.

Reply:
@adongare here are some example errors from errorExceptions: 401 : [no body], Could not resolve identifier AdultOutpatientEncounters in the current library. Member Qualifying Encounters not found for type null. Could not determine signature for invocation of operator System.Exists....I think some are syntax errors some are errors from global library?? Anyway, these errors are ignored by frontend, frontend only deals externalErrors, and parsing errors as well.

Reply:
Go it, thanks for the explanation.

Reply:
@jkotanchik-SB should we not consider all the errors except warnings? syntax errors or errros from included library makes the CQL invalid as well and they need to know, right?

Reply:
At this point in the import flow, is the CQL translated by MADiE and the Libraries in MADiE?

If so, then there _shouldn't_ be any errors in the included libraries as they must be versioned to be used in translation. "Shouldn't" meaning there is an edge case here if the translator is feed a non-versioned Library.

Reply:
@jkotanchik-SB @adongare It looks what the transfer measure is doing is the same as creating a measure, it does get the elmJson from translator, and then check if there is error. I do see errorExceptions but it looks like they are ignored by the frontend editor, only externalErrors are considered an error.